frontend/src/components/TicketBoard/index.js

```diff
@@ -9,25 +9,108 @@ import {
         <div
-          className="task"
+          className="task-card-container"
           {...provided.draggableProps}
           {...provided.dragHandleProps}
           ref={provided.innerRef}
           onClick={() => {
             dispatch({ type: SET_ACTIVE_TICKET, payload: props.task });
           }}
         >
-          {props.task.code}
-          <br />
-          {props.task.title}
+          <div className="task-card-color-tab" />
+          <div className="task-card-content">
+            <div className="task-card__header">
+              <div className="task-card__code">{props.task.code}</div>
+              <div
+                className={clsx(
+                  isReminder && "task-card__reminder--notify-true",
+                  !isReminder && "task-card__reminder--notify-false"
+                )}
+              >
+                <div className="task-card__notificaton-container">
+                  <NotificationIcon
+                    color={isReminder ? "#FFFFFF" : "#000000"}
+                    className="notification-icon"
+                    width={25}
+                    height={20}
+                  />
+                </div>
+              </div>
+            </div>
+            <div className="task-card__body">
+              <div className="task-card__title">{props.task.title}</div>
+            </div>
+            <div className="task-card__footer">
+              {completedSubtasks > 0 && (
+                <div className="task-card__subtasks-container">
+                  <CheckBoxIcon
+                    className={"task-card__subtask-checkbox"}
+                    checkColor={"#E4E5EA"}
+                    boxColor={"#FFFFFF"}
+                    width={14}
+                    height={14}
+                  />
+                  <div className="task-card__subtasks-completed-text">{`${completedSubtasks} / ${totalSubtasks} `}</div>
+                </div>
+              )}
+              <div className="task-card__assignees-container">
+                {assignees.map((assignee) => {
+                  const assigneeId = getColorNum(assignee.id);
+                  const colorNumberMod =
+                    assigneeId % Object.keys(colors).length;
+                  const assigneeColor = colors[colorNumberMod];
+                  const assigneeInitials = `${assignee.firstName[0].toUpperCase()}${assignee.lastName[0].toUpperCase()}`;
+                  return (
+                    <div
+                      key={assignee.id}
+                      className="task-card__assignees-container"
+                    >
+                      <AssigneeIcon
+                        shapeColor={assigneeColor}
+                        textColor={"white"}
+                        className="task-card__assignee-container"
+                        label={assigneeInitials}
+                      />
+                      {/* <div className="task-card__assignee-initials">AT</div> */}
```

Comment:
The way I tried to implement this was that I had a div that had the coloured bubble image in the back and then I tried to layer it with the div initials element.  For the initials element I tried to style it so that this element would be position absolute but it seemed a little janky to use and I am afraid it would not scale or position properly when in different screen sizes. 

I also tried layering text inside the SVG in the assignee icon as well but for some reason that did not seem to work because the text would always be behind the bubble. For some reason the rendering of SVGs is a little odd and the text does not really layer well when in an SVG. 

Perhaps there is another way we could do this?